### PR TITLE
Update Iot Headed/Headless API ref

### PR DESCRIPTION
**Pull request by sgchoi5**
- Target: 4.0 / 5.0 / 5.5
- Removed unnecessary modules
- Fixed some minor issues (broken link, etc)

### CODE REVIEW

File: docs/application/native/api/iot-headed/4.0/index.html
Comment (mijin85cho) [inline docs/application/native/api/iot-headed/4.0/index.html:109]: Just a question.

IoT Headed / Headless API are based on Tizen Native API. Is it the reason of keep using the "Tizen native API" in the pages?



image

**Review by mijin85cho — COMMENTED** [review]
Hello.

I skimmed the pages and got below queries.

Also, I reported an issue with the updated pages. 

With the updated files, the LNB works.
However, in some pages, the layout is dragged down to the end of LNB, so at first sight the page shows just a blank window.

image

After scrolling : 
image


I reported this issue to  https://github.sec.samsung.net/RS8-TIZENWEBSITE/docs.tizen.org/issues/430.
Would you also please check the issue page to get the status?

File: docs/application/native/api/iot-headed/4.0/index.html
Comment (sgchoi5) [reply docs/application/native/api/iot-headed/4.0/index.html:109]: Iot Profiles support Native API. but, different coverage compared with Mobile / Wearable.

File: docs/application/native/api/iot-headed/4.0/classDali_1_1Toolkit_1_1Button.html
Comment (sgchoi5) [reply docs/application/native/api/iot-headed/4.0/classDali_1_1Toolkit_1_1Button.html:81]: DALi is deprecated. They have stopped the maintenance of DALi...

File: docs/application/native/api/iot-headed/4.0/ecore_thread_example_c.html
Comment (sgchoi5) [reply docs/application/native/api/iot-headed/4.0/ecore_thread_example_c.html:217]: I think .. new version is correct, because previous 4.0 headed api ref was made by very old header files.

File: docs/application/native/api/iot-headed/4.0/ecore_thread_example_8c-example.html
Comment (sgchoi5) [reply docs/application/native/api/iot-headed/4.0/ecore_thread_example_8c-example.html:504]: I think .. new version is correct, because previous 4.0 headed api ref was made by very old header files.

File: docs/application/native/api/iot-headed/4.0/ecore_thread_example_8c-example.html
Comment (sgchoi5) [reply docs/application/native/api/iot-headed/4.0/ecore_thread_example_8c-example.html:446]: I think .. new version is correct, because previous 4.0 headed api ref was made by very old header files.

Comment (mijin85cho) [reply docs/application/native/api/iot-headed/4.0/ecore_thread_example_8c-example.html:446]: It is same in https://docs1.stg.tizen.org/staging/1358/application/native/api/iot-headed/6.0/ecore_thread_example_8c-example.html, so I asked.

image

However if you think this issue is not critical, I'll just merge this PR after we get the reply for the dragged window issue.

File: docs/application/native/api/iot-headed/4.0/classDali_1_1Toolkit_1_1Button.html
Comment (mijin85cho) [reply docs/application/native/api/iot-headed/4.0/classDali_1_1Toolkit_1_1Button.html:81]: Question. The issue I reported about the window display is happening in DALi.

If DALi is deprecated, can we just ignore that issue on https://github.sec.samsung.net/RS8-TIZENWEBSITE/docs.tizen.org/issues/430?

In other pages I couldn't find the same issue yet.

Comment (sgchoi5) [reply docs/application/native/api/iot-headed/4.0/classDali_1_1Toolkit_1_1Button.html:81]: I think.. if it is not critical issue, hard to fix DALi issues... : (

File: docs/application/native/api/iot-headed/4.0/ecore_thread_example_8c-example.html
Comment (sgchoi5) [reply docs/application/native/api/iot-headed/4.0/ecore_thread_example_8c-example.html:446]: In case of EFL, I will try to fix issues in this page. 
https://code.sec.samsung.net/confluence/pages/viewpage.action?pageId=194846376

Hard to fix it quickly, because EFL team needs to fix upstream header first and sync it to Tizen Platform and SR should be completed. It takes ... at least 2~3 weeks.

**Comment by mijin85cho** [comment]
@sgchoi5 I found some issues within this PR, 
however most of them are same with the current published version, or not critical, or will take some time to fix (or hard to fix).

So, I'd like to merge this PR for now. 
If so, about weeks later you can create a new PR for new version that will be generated at that time. Will you?

File: docs/application/native/api/iot-headed/4.0/ecore_thread_example_8c-example.html
Comment (sgchoi5) [reply docs/application/native/api/iot-headed/4.0/ecore_thread_example_8c-example.html:446]: Yes, correct. I am trying to fix them all.. t.t 
Yesterday, I sent all issues to developers. waiting for fix and feedback..

**Comment by sgchoi5** [comment]
Yes, I will try. but, not easy to fix issues, because developers must update their header files.

File: docs/application/native/api/iot-headed/5.0/group__Evas__Main__Group.html
Comment (mijin80lee) [inline docs/application/native/api/iot-headed/5.0/group__Evas__Main__Group.html:100]: Need to check the function 'evas_init (void)' compared to mobile API Reference.
(ref. https://docs.tizen.org/application/native/api/mobile/6.0/group__Evas__Main__Group.html : EINA_DEPRECATED int)

File: docs/application/native/api/iot-headed/5.0/group__CAPI__OAUTH2__MODULE.html
Comment (mijin80lee) [inline docs/application/native/api/iot-headed/5.0/group__CAPI__OAUTH2__MODULE.html:117]: Could you delete `a&gt;` after `<b>Feature List</b>.</a>`?
The 6.0 version also has the same content, please modify it.

File: docs/application/native/api/iot-headed/5.0/group__ATSPI.html
Comment (mijin80lee) [inline docs/application/native/api/iot-headed/5.0/group__ATSPI.html:93]: Check the Menu tree and confirm exact location.
The 6.0 version also has the same issue, please modify it.

image

**Comment by mijin80lee** [comment]
Could you change URLs of 'Feature Element' of following pages as below?
(Tizen Docs, https://docs.tizen.org/application/tizen-studio/native-tools/manifest-text-editor#feature-element)

docs/application/native/api/iot-headed/6.0/group__CAPI__SYSTEM__PERIPHERAL__IO__ADC__MODULE.html
docs/application/native/api/iot-headed/6.0/group__CAPI__SYSTEM__PERIPHERAL__IO__GPIO__MODULE.html
docs/application/native/api/iot-headed/6.0/group__CAPI__SYSTEM__PERIPHERAL__IO__I2C__MODULE.html
docs/application/native/api/iot-headed/6.0/group__CAPI__SYSTEM__PERIPHERAL__IO__PWM__MODULE.html
docs/application/native/api/iot-headed/6.0/group__CAPI__SYSTEM__PERIPHERAL__IO__SPI__MODULE.html
docs/application/native/api/iot-headed/6.0/group__CAPI__SYSTEM__PERIPHERAL__IO__UART__MODULE.html
docs/application/native/api/iot-headed/6.0/group__CAPI__SYSTEM__UPDATE__CONTROL__MODULE.html

**Comment by mijin80lee** [comment]
Could you add '(Deprecated)' to the title on DALi Toolkit page? 
docs/application/native/api/iot-headed/6.0/group__dali__toolkit.html

image

File: docs/application/native/api/iot-headed/5.0/group__Evas__Main__Group.html
Comment (sgchoi5) [reply docs/application/native/api/iot-headed/5.0/group__Evas__Main__Group.html:100]: It is wrong deprecated TAG. I've added an issue to https://code.sec.samsung.net/confluence/pages/viewpage.action?pageId=194846376
Issue No. 60

EFL part need to update header file.... Can't fix it right now.

Comment (mijin80lee) [reply docs/application/native/api/iot-headed/5.0/group__Evas__Main__Group.html:100]: Okay, I got it.

File: docs/application/native/api/iot-headed/5.0/group__CAPI__OAUTH2__MODULE.html
Comment (sgchoi5) [reply docs/application/native/api/iot-headed/5.0/group__CAPI__OAUTH2__MODULE.html]: oauth2_doc.h should be fixed to remove unnecessary a&gt;
I've added it to issue page with No.59.

In this case, I've fixed html directly and pushed changes.

File: docs/application/native/api/iot-headed/5.0/group__ATSPI.html
Comment (sgchoi5) [reply docs/application/native/api/iot-headed/5.0/group__ATSPI.html:93]: Issue. 61 (https://code.sec.samsung.net/confluence/pages/viewpage.action?pageId=194846376)
need to fix EFL header.

**Comment by sgchoi5** [comment]
> Could you add '(Deprecated)' to the title on DALi Toolkit page?
> docs/application/native/api/iot-headed/6.0/group__dali__toolkit.html
> 
> image

will be fixed with next update. Recently, Dali team fixed it.

**Comment by sgchoi5** [comment]
> https://docs.tizen.org/application/tizen-studio/native-tools/manifest-text-editor#feature-element

I've made a new page for this issue. Just, sent a request email to developers.
https://code.sec.samsung.net/confluence/display/API/Native+API+Ref%3A+developer.tizen.org+%3D%3E+docs.tizen.org

**Review by mijin85cho — APPROVED** [review]
Pending issues will be checked with next version.

**Pull request merged**
